extensions/interactions/TextInput/TextInput.html

```diff
@@ -9,9 +27,12 @@
   <div class="row text-input-container">
     <form ng-submit="submitAnswer(answer)" role="form" class="form-horizontal">
       <div class="col-lg-12 col-md-12 col-sm-12">
+        <input ng-hide="rows > 1" type="text" ng-model="answer" class="form-control" placeholder="{{placeholder}}" autofocus/>
```

Comment:
Also, please add the attribute 

```
    focus-on="<[labelForFocusTarget]>"
```

to this field. This will enable focus to be set to it once it appears on the page. (Note that this is a custom thing we built for Oppia; it isn't general html.)

 Also also -- note that the 'input' tag represent a void element and doesn't need to be closed. That is, you can change "/>" to ">".